src/js/components/Video/StyledVideo.js

```diff
@@ -35,6 +35,14 @@ const StyledVideoContainer = styled.div`
   overflow: hidden;
   position: relative;
   ${genericStyles};
+  &:focus {
+    outline-offset: 0px;
+    outline: ${(props) => {
+      const color = normalizeColor('focus', props.theme);
+      if (color) return `2px solid ${color};`;
```

Comment:
Should this come from utils/styles.js focusStyles?

Reply:
Yes, I will go back and change that